compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/test/decompose_im2col.mlir

```diff
@@ -0,0 +1,73 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(func.func(iree-linalg-ext-decompose-im2col))" --split-input-file %s | FileCheck %s
+
+#map = affine_map<(d0) -> (d0 * 4)>
+module {
+  func.func @im2col(%arg0: tensor<2x34x34x640xf32>, %m_size: index, %m_off: index, %k: index) -> tensor<2x?x4xf32> {
```

Comment:
If I understand correctly, this test does not tile `k` right? In this context, I think we can rename `im2col` to `im2col_untile_k`, which looks better to me.